seeder/src/main/interfaces/express_urls.ts

```diff
@@ -0,0 +1,15 @@
+import { get_free_port } from "$utils/port";
+
+export const express_port = await get_free_port();
+
+const base_url = `http://localhost:${express_port}`;
+
+export type ExpressUrls = {
+	shiinobi_healthcheck: string;
+	staff: string;
+};
```

Comment:
Could we remove this?

Reply:
And? declare inside `d.ts` itself? uh might be okay since its a declaration only place.

Reply:
>And? declare inside d.ts itself? uh might be okay since its a declaration only place.

Make it dynamic, so it automatically picks up the fields on object

Reply:
> > And? declare inside d.ts itself? uh might be okay since its a declaration only place.
> 
> Make it dynamic, so it automatically picks up the fields on object

Uh dont get it? dynamic? automatic pick up?

Reply:
https://github.com/coreproject-moe/CoreProject/pull/3996/commits/7a6b7299540f3169dbe0621a643ada5f5ce6d33e
(still requires relative import)